### ISSUE TITLE
Allow different base url depending on the request

### ISSUE THREAD

**Issue by filipeximenes** [vintasoftware/tapioca-wrapper#46]
One possible solution:
- Allow specifying a different root url for a resource.
```
'resource_example': {
    'resource': '{id}',
    'root_url': 'https://api.lalala.com/',
    'docs': '...docs url...'
}
```
the resulting url will then be: ``https://api.lalala.com/{id}``

**Comment by filipeximenes** [comment]
Discussed elsewhere.
We will pass ```request_method``` to ```get_request_kwargs```.
https://github.com/vintasoftware/tapioca-wrapper/blob/master/tapioca/tapioca.py#L139

**Comment by filipeximenes** [comment]
Closed on #48
